eng/tools/dependency-testing/index.js

```diff
@@ -332,7 +332,13 @@ async function main(argv) {
   const testFolder = argv["test-folder"];
   const dryRun = argv["dry-run"];
 
-  const packageName = artifactName.replace("azure-", "@azure/");
+  let packageName = "";
```

Comment:
We should try using a regex here or a more general replace pattern.

Reply:
Or move this parsing further into the getPackageFromRush function and do the similar normalizing you are doing in other places.